### ISSUE TITLE
Defining 'cls.syntax' has no effect anymore

### ISSUE THREAD

**Issue by TomHarrop** [ckaznocha/SublimeLinter-contrib-write-good#14]
Hi,

I get the following error when opening SublimeText. I guess it relates to #11. I can try a PR if anyone else is having this problem?

Thanks!

```
SublimeLinter: ERROR:
=====================

writegood: Defining 'cls.syntax' has no effect anymore. Use http://www.sublimelinter.com/en/stable/linter_settings.html#selector instead.


SublimeLinter: ERROR:
=====================

writegood: Defining 'cls.selectors' has no effect anymore. Use http://www.sublimelinter.com/en/stable/linter_settings.html#selector instead.


SublimeLinter: ERROR:
=====================

writegood disabled. 'cls.defaults' is mandatory and MUST be a dict.
```

**Comment by ChrisCummins** [comment]
I have submitted a PR for this. In the meantime you can use by manually installing:

```sh
$ git clone https://github.com/ChrisCummins/SublimeLinter-contrib-write-good.git > ~/.subl/Packages/SublimeLinter-contrib-write-good
```

**Comment by TomHarrop** [comment]
@ChrisCummins Thanks for fixing this!

I couldn't get the linter from your repo to install by cloning it into my config directory.

Instead, I forked your repo, merged your branch into master in my fork, then added my fork to Sublime Package Control using Package Control: Add Repository. Then I was able to install SublimeLinter-contrib-write-good via Package Control: Install Package.

Since *this* repo seems to be no longer active, if you want to merge your branch to master in your repo, I'll delete my fork.

**Comment by dillonma** [comment]
Just seeing this now. Thanks @ChrisCummins -- any chance yours could go in the official repository? Happy to do a git clone for now ...

**Comment by ckaznocha** [comment]
Hey @TomHarrop or @dillonma or anyone else following along. Thanks for reporting and working on these issues. As you've noticed I don't actively maintain this anymore. I no longer use ST and haven't had the time to keep this project up to date but I'd really hate to see it die because I know there are people out there that get a lot of use out of it. Would you like to be added to this repo as a maintainer? If not, I'll put out an issue as a call for maintainers or look into transferring it to the SublimeLnters org

**Comment by ChrisCummins** [comment]
Hi @ckaznocha, thanks for getting back to us! I appreciated that you’ve moved on to other things. I’m happy to maintain this project, it shouldn’t need a lot of work to keep it going.

Cheers,
Chris

**Comment by TomHarrop** [comment]
@ChrisCummins gets my vote, and I'm also happy to help however I can.

**Comment by dillonma** [comment]
@ChrisCummins gets my vote as well, thanks @ckaznocha for replying and making sure this project can keep going!

**Comment by ChrisCummins** [comment]
Thanks @ckaznocha I now have push permissions for this repo. At the weekend I plan to first to fix the Travis config, then get a fix for this issue merged and figure out how to update the Package Control package for an easy install. Will ping y'all here when it's done

**Comment by ChrisCummins** [comment]
Fixed as of #15, version 2.2.1.
